helpers/calendar/push_notifications.py

```diff
@@ -0,0 +1,52 @@
+import graphene
+from helpers.calendar.credentials import Credentials
+from flask import jsonify
+from pyfcm import FCMNotification
+import uuid
+import os
+
+notification_url = (os.getenv('NOTIFICATION_URL')
+                    if os.getenv('APP_SETTINGS') == 'production'
+                    else os.getenv('DEV_NOTIFICATION_URL'))
+
+push_service = FCMNotification(api_key=os.getenv('FCM_API_KEY'))
+
+
+class NotificationChannelPerRoom(graphene.ObjectType):
+    expiration = graphene.String()
+    id = graphene.String()
+    kind = graphene.String()
+    resourceId = graphene.String()
+    resourceUri = graphene.String()
+
+
+class PushNotification():
+
+    def create_channels(self, all_rooms):
+        request_body = {
+            "id": None,
+            "type": "web_hook",
+            "address": notification_url
+        }
+        service = Credentials.set_api_credentials(self)
+        channels = []
+        for room in all_rooms:
+            print(notification_url)
```

Comment:
@omobosteven, Great work on this task, kindly explain why you have a Print statement here?

Reply:
@RachaelNantale, That was a mistake. I'm taking it out.